jooq-pg/src/main/resources/db/migration/ehr/V15__data_tables_add_parent_num.sql

```diff
@@ -16,137 +16,240 @@
-        WHERE ch.ehr_id=pa.ehr_id
-          AND ch.entity_idx_len > 1
-          AND pa.entity_idx_len = ch.entity_idx_len - 1
-          AND ch.entity_idx ^@ pa.entity_idx
-          AND ch.parent_num = 0;
-        UPDATE ehr_folder_data_history ch SET parent_num=pa.num
-        FROM ehr_folder_data_history pa
-        WHERE ch.ehr_id=pa.ehr_id
-          AND ch.sys_version=pa.sys_version
-          AND ch.entity_idx_len > 1
-          AND pa.entity_idx_len = ch.entity_idx_len - 1
-          AND ch.entity_idx ^@ pa.entity_idx
-          AND ch.parent_num = 0;
-
-        -- num cap
-        UPDATE comp_data pa SET num_cap = (
-            select max(ch.num) FROM comp_data ch
-            WHERE ch.vo_id=pa.vo_id
-              AND pa.entity_idx_len <= ch.entity_idx_len
-              AND ch.entity_idx ^@ pa.entity_idx
-        ) WHERE pa.num_cap = -1;
-        UPDATE comp_data_history pa SET num_cap = (
-            select max(ch.num) FROM comp_data_history ch
-            WHERE ch.vo_id=pa.vo_id
-              AND ch.sys_version=pa.sys_version
-              AND pa.entity_idx_len <= ch.entity_idx_len
-              AND ch.entity_idx ^@ pa.entity_idx
-        ) WHERE pa.num_cap = -1;
-        UPDATE ehr_status_data pa SET num_cap = (
-            select max(ch.num) FROM ehr_status_data ch
-            WHERE ch.ehr_id=pa.ehr_id
-              AND pa.entity_idx_len <= ch.entity_idx_len
-              AND ch.entity_idx ^@ pa.entity_idx
-        ) WHERE pa.num_cap = -1;
-        UPDATE ehr_status_data_history pa SET num_cap = (
-            select max(ch.num) FROM ehr_status_data_history ch
-            WHERE ch.ehr_id=pa.ehr_id
-              AND ch.sys_version=pa.sys_version
-              AND pa.entity_idx_len <= ch.entity_idx_len
-              AND ch.entity_idx ^@ pa.entity_idx
-        ) WHERE pa.num_cap = -1;
-        UPDATE ehr_folder_data pa SET num_cap = (
-            select max(ch.num) FROM ehr_folder_data ch
-            WHERE ch.ehr_id=pa.ehr_id
-              AND pa.entity_idx_len <= ch.entity_idx_len
-              AND ch.entity_idx ^@ pa.entity_idx
-        ) WHERE pa.num_cap = -1;
-        UPDATE ehr_folder_data_history pa SET num_cap = (
-            select max(ch.num)
-             FROM ehr_folder_data_history ch
-             WHERE ch.ehr_id=pa.ehr_id
-               AND ch.sys_version=pa.sys_version
-               AND pa.entity_idx_len <= ch.entity_idx_len
-               AND ch.entity_idx ^@ pa.entity_idx
-        ) WHERE pa.num_cap = -1;
-    end if;
-
-end $$;
-
-DROP INDEX IF EXISTS mig_ehr_folder_data_history_num_idx;
-DROP INDEX IF EXISTS mig_ehr_folder_data_num_idx;
-DROP INDEX IF EXISTS mig_ehr_status_data_history_num_idx;
-DROP INDEX IF EXISTS mig_ehr_status_data_num_idx;
-DROP INDEX IF EXISTS mig_comp_data_history_num_idx;
-DROP INDEX IF EXISTS mig_comp_data_num_idx;
-
-DROP INDEX IF EXISTS comp_data_idx;
-DROP INDEX IF EXISTS comp_data_leaf_idx;
-DROP INDEX IF EXISTS ehr_status_data_idx;
+--
+-- Migrate
+--
+
```

Comment:
The migration indexes are missing

Reply:
now I'm confused -> https://github.com/ehrbase/ehrbase/pull/1389#discussion_r1718247751